test/parallel/test-event-emitter-errors.js

```diff
@@ -7,4 +7,4 @@ var EE = new EventEmitter();
 
 assert.throws(function() {
   EE.emit('error', 'Accepts a string');
-}, /Accepts a string/);
+}, /Unhandled "error" event. \(Accepts a string\)/);
```

Comment:
I'd opt to just not try to print the error. You could use `util.format()`, but it prints the whole stack trace for errors, so it's kind of loud.